specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Devices/IotHub.tsp

```diff
@@ -93,7 +93,8 @@ model DeviceConnectionStateEventProperties {
 /** Schema of the Data property of an EventGridEvent for a device telemetry event (DeviceTelemetry). */
 model DeviceTelemetryEventProperties {
   /** The content of the message from the device. */
-  body: Record<{}>;
+  #suppress "@azure-tools/typespec-azure-core/bad-record-type" "The type of body is object"
+  body: Record<unknown>;
```

Comment:
Is this based off the examples?

Reply:
so I based this on that the type in the swagger is just object without additionalProperties specified -- let me check the examples

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/548a10278921a7ce1de668228b187f0dfa4a5be8/specification/eventgrid/data-plane/Microsoft.Devices/stable/2018-01-01/examples/cloud-events-schema/device_telemetry.json#L8